src/demo/pages/BlockchainTransaction/SignTransactionSendPayment.tsx

```diff
@@ -48,20 +48,15 @@ const SignTransactionSendPayment = (props: SignTransactionSendPaymentProps) => {
       const accountName = await user?.getAccountName();
 
       if (accountName) {
-        let accountBalance = await eosioTokenContract.getBalance(accountName);
+        const accountBalance = await eosioTokenContract.getBalance(accountName);
 
         props.setBalance(accountBalance);
-        setAmount(Math.floor(accountBalance / 2));
         if (accountBalance > 10) return;
-
         await user?.signTransaction("eosio.token", "selfissue", {
           to: accountName,
           quantity: "10 SYS",
           memo: "test",
         });
-        accountBalance = accountBalance + 10;
```

Comment:
similar here. This should stay. the selfissue transaction is used in case they dont have enough tokens. If we give them more tokens, then we also need to update their balance.